### ISSUE TITLE
Feature Issue: Selecting Days

### ISSUE THREAD

**Issue by kraymitchell** [felicegattuso/datedropper#17]
While the day selector works pretty well, a few things I noticed that will cause issues on Mobile (Via iOS8 on iPhone6

1. It doesn't seem to go back to the start, so if I'm on the 29th, I can't just go through to the 1st, I have to go backwards, which takes a long time.

2. Perhaps an option where you double tap brings up the full day calendar to select a day for those that don't want to scroll through all the numbers.

I like where this is going, but I think some more thought on mobile UX is needed to make it awesome.

Keep up the great work.

**Comment by coRz** [comment]
Yeah I noticed that too, the part where you cannot go back to the start gonna work on it.
